neqo-http3/src/client_events.rs

```diff
@@ -44,19 +46,23 @@ pub struct Http3ClientEvents {
 }
 
 impl Http3ClientEvents {
-    pub fn header_ready(&self, stream_id: u64) {
+    /// Add a new `HeaderReady` event.
+    pub(crate) fn header_ready(&self, stream_id: u64) {
```

Comment:
Are these still `pub(crate)` because the type is being exported outside of neqo?

Reply:
yes, this as setters and they are only set by code in neqo-http3